roles/prereqs/prerequisite/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+
+prerequisite_package_name: ''
```

Comment:
The role prereqs/package is currently being used by the prereqs/libsemanagepython